File: 08 - ClassPhotos/08.py
def classPhotos(redShirtHeights, blueShirtHeights):
    r, b = [redShirtHeights, blueShirtHeights]
    r.sort()
    b.sort()
    r_counter, b_counter = [0, 0]

    for i in range(len(r)):
        if r[i] == b[i]:
            return False
        if r[i] > b[i]:
            r_counter += 1
        elif b[i] > r[i]:
            b_counter += 1

    return r_counter == len(r) or b_counter == len(r)

File: 08 - ClassPhotos/08.js
function classPhotos(redShirtHeights, blueShirtHeights) {
    let r = redShirtHeights.slice(); // Create a copy to avoid mutating the original array
    let b = blueShirtHeights.slice(); // Create a copy to avoid mutating the original array
  
    r.sort((a, b) => a - b);
    b.sort((a, b) => a - b);
  
    let rCounter = 0;
    let bCounter = 0;
  
    for (let i = 0; i < r.length; i++) {
      if (r[i] === b[i]) return false
      if (r[i] > b[i]) {
        rCounter++;
      } else if (b[i] > r[i]) {
        bCounter++;
      }
    }
  
    return rCounter === r.length || bCounter === r.length;
  }